docs/var.tfvars-doc.md

```diff
@@ -51,6 +51,15 @@ worker                      = {instance_type    = "<worker-compute-template>", i
 You can optionally set worker `count` value to 0 in which case all the cluster pods will be running on the master/supervisor nodes. 
 Ensure you use proper sizing for master/supervisor nodes to avoid resource starvation for containers.
 
+`availability_zone` is an optinal attribute for bastion, bootstrap, master and worker. If it is specified, the VM will be created in the specified `availability_zone`, otherwise value of `openstack_availability_zone` will be used. 
```

Comment:
nit: optinal => optional

Reply:
Also can you please include the `Fix #179` in the commit message.

Reply:
Done